user-data-store-server/src/test/java/com/wultra/security/userdatastore/restclient/AttachmentRestClientTest.java

```diff
@@ -56,7 +58,8 @@ class AttachmentRestClientTest {
     private UserDataStoreRestClient restClient;
 
     @BeforeAll
-    void initRestClient() throws Exception {
+    void initTests() throws Exception {
+        Security.addProvider(new BouncyCastleProvider());
```

Comment:
We may move the initialization of the provider into a static constructor in the SpringBootApplication class.

Reply:
Ok, let's handle this separately, if required.